app/factories/bulkrax/object_factory.rb

```diff
@@ -227,12 +243,32 @@ def collection_type(attrs)
     def transform_attributes(update: false)
       @transform_attributes = attributes.slice(*permitted_attributes)
       @transform_attributes.merge!(file_attributes(update_files)) if with_files
+      @transform_attributes = remove_blank_hash_values(@transform_attributes)
       update ? @transform_attributes.except(:id) : @transform_attributes
     end
 
     # Regardless of what the Parser gives us, these are the properties we are prepared to accept.
     def permitted_attributes
-      klass.properties.keys.map(&:to_sym) + %i[id edit_users edit_groups read_groups visibility work_members_attributes admin_set_id]
+      klass.properties.keys.map(&:to_sym) + base_permitted_attributes
+    end
+
+    # Return a copy of the given attributes, such that all values that are empty or an array of all
+    # empty values are fully emptied.  (See implementation details)
+    #
+    # @param attributes [Hash]
+    # @return [Hash]
+    #
+    # @see https://github.com/emory-libraries/dlp-curate/issues/1973
+    def remove_blank_hash_values(attributes)
+      dupe = attributes.dup
+      dupe.each do |key, values|
+        if values.is_a?(Array) && values.all? { |value| value.is_a?(String) && value.empty? }
+          dupe[key] = []
+        elsif values.is_a?(String) && values.empty?
+          dupe[key] = nil
+        end
+      end
+      dupe
```

Comment:
will this interfere with the intent of pr https://github.com/samvera-labs/bulkrax/pull/374?

Reply:
That's a good question, and I don't know the answer.  My cursory read is that there might be a conflict.  But we'll need to discuss the spec.

@bwatson78 I'd love for you to weigh in to help us triangulate on approaches.

Reply:
@jeremyf this [slack convo](https://assaydepot.slack.com/archives/G0313NK5NMA/p1673638808870859) may also help us. in the event that this code does make #374 irrelevant, there's still a way to delete data with bulkrax. however...it would require updating users on how deleting is done.

Reply:
In addition to @alishaevn's question, there is [this section in the Wiki](https://github.com/samvera-labs/bulkrax/wiki/Interacting-with-Metadata) that I think has extra context; it lays out how Bulkrax should interpret empty cells, the absence of columns, etc. I think we should either: 

1. Make sure the behavior still matches the Wiki, as that's what is intended / what users expect, or 
2. Update Bulkrax's opinions (including the Wiki article) as to how we want to handle these use cases

Reply:
@jeremyf I apologize for not responding sooner. I see that this is already merged. If the question still stands, let me know.